uui-editor/src/plugins/imagePlugin/ImageElement.tsx

```diff
@@ -47,21 +47,27 @@ export function ImageElement({
         focused && selected && css.resizeHandleVisible, // for mobile
     ];
 
+    const imageParentRef = useRef(null);
+
     useMediaState();
 
     const [currentWidth] = useResizableStore().use.width();
     const isCaptionEnabled = useMemo(() => {
         let captionEnabled = false;
         if (currentWidth && typeof currentWidth === 'number') {
             captionEnabled = currentWidth >= MIN_CAPTION_WIDTH;
+        } else if (currentWidth && (typeof currentWidth === 'string')) {
```

Comment:
Can we have the same logic for both cases, for string and number? I guess that it's doesn't matter which type of `currentWidth`, it's shouldn't figure out in this logic at all. We always need to have valid `currentWidth` value in editor state and always rely on it. As i understand, the problem here that we don't have correct `currentWidth` right after the initialization, i suggest to add logic which will set it.